log_init.py

```diff
@@ -64,17 +64,21 @@ def initialize_logging():
         filemode="w",  # Log everything in the file
         level=logging.DEBUG,
     )
-    # Apply DeltaTimeFormatter formatter to file output
-    logging.getLogger("").root.handlers[0].setFormatter(file_formatter)
+    # Apply non-colored formatter to file output
+    logging.getLogger("").root.handlers[0].setFormatter(bw_log_formatter)
 
     # Get the visible log level for the console logger from config.yaml
     config_data = config.open_config()
     console_log_level = config_data.get("verbosity", "DEBUG")
+    color_log = config_data.get("color_log", False)
 
     # Set up the console logger
     console = logging.StreamHandler()
     console.setLevel(console_log_level)  # Log the appropriate information to console
-    console.setFormatter(formatter)  # Apply DeltaTimeFormatter formatter
+    if color_log:
```

Comment:
Small nit-pick:

```python 
formatter_to_use = bw_log_formatter
if color_log:
    formatter_to_use = color_log_formatter
console.setFormatter(formatter_to_use)
```

It's slightly less nested and only calls `console.setFormatter` the one time (rather than once in two different branches of code).

I also think it's easier to read (less nested things tend to be easier to read)

Reply:
That's fair, I'll tweak.